packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -390,6 +390,11 @@ export const selectCoinjoinAccountByKey = createSelector(
     (accounts, accountKey) => accounts.find(account => account.key === accountKey),
 );
 
+export const selectSessionByAccountKey = createSelector(
+    [selectCoinjoinAccounts, (_state: CoinjoinRootState, accountKey: string) => accountKey],
+    (accounts, accountKey) => accounts.find(account => account.key === accountKey)?.session,
```

Comment:
Results of input selectors are === same even if session has been changed, so output selector is not executed again and cached result is returned

Reply:
I tried using JSON compare in the selector and it didn't help :( But perhaps it doesn't affect `selectCoinjoinAccounts`? Or what do you mean?